packages/ui/src/ui/pages/query-tracker/QueryACO/QueryACOSelect/index.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import {Select} from '@gravity-ui/uikit';
+import {useCallback} from 'react';
+import {useQueryACO} from '../useQueryACO';
+
+import './QueryACOSelect.scss';
+export const QueryACOSelect: React.FunctionComponent<{}> = () => {
+    const {selectACOOptions, isFlight, changeDraftQueryACO, currentDraftQueryACO} = useQueryACO();
+
+    const handleACOChange = useCallback(
+        (value: string[]) => {
+            const aco = value[0];
+
+            return changeDraftQueryACO({aco});
+        },
+        [changeDraftQueryACO],
+    );
+
+    return (
+        <Select<string>
+            filterable
+            disabled={isFlight}
+            width={'auto'}
+            pin="clear-round"
+            label="Query access:"
+            className={'query-aco-select'}
+            options={selectACOOptions}
+            value={[currentDraftQueryACO]}
+            onUpdate={handleACOChange}
+        />
```

Comment:
Let's additionally provide edit link for selected ACO:
```tsx
<Link
    target="_blank"
    url={genNavigationUrl(
         queryTrackerInfo.cluster_name, 
        `//sys/access_control_object_namespaces/queries/${currentDraftQueryACO}`
    )}
>
    Edit {acoName}
    <Icon awesome="external-link"/>
</Link>
```
Maybe we don't need to show the link for 'nobody' item